From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: mrvictory1 <108172910+mrvictory1@users.noreply.github.com>
Date: Wed, 1 Jul 2026 14:45:20 +0300
Subject: [PATCH 04/13] HACK: comment out mallopt on loader.c

---
 dlls/ntdll/unix/loader.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/dlls/ntdll/unix/loader.c b/dlls/ntdll/unix/loader.c
index ae68f38909b..3a4f885e4ae 100644
--- a/dlls/ntdll/unix/loader.c
+++ b/dlls/ntdll/unix/loader.c
@@ -2453,7 +2453,7 @@ static void start_main_thread(void)
     set_thread_teb( teb );
 #endif
 
-    mallopt( M_PERTURB, 0xff );
+    // mallopt( M_PERTURB, 0xff );
     init_startup_info();
     *(ULONG_PTR *)&peb->CloudFileFlags = get_image_address();
     set_load_order_app_name( main_wargv[0] );
@@ -2462,7 +2462,7 @@ static void start_main_thread(void)
     load_ntdll();
     load_wow64_ntdll( main_image_info.Machine );
     load_apiset_dll();
-    mallopt( M_PERTURB, 0 );
+    // mallopt( M_PERTURB, 0 );
     server_init_process_done();
 }
 
-- 
2.39.5 (Apple Git-154)

