From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: mrvictory1 <108172910+mrvictory1@users.noreply.github.com>
Date: Wed, 1 Jul 2026 19:44:32 +0300
Subject: [PATCH 08/13] Undo fsync disable on non-Linux OS, define __NR_futex

---
 dlls/ntdll/unix/fsync.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/dlls/ntdll/unix/fsync.c b/dlls/ntdll/unix/fsync.c
index cd214fa141f..874ea820057 100644
--- a/dlls/ntdll/unix/fsync.c
+++ b/dlls/ntdll/unix/fsync.c
@@ -22,7 +22,6 @@
 #if 0
 #pragma makedep unix
 #endif
-#ifdef __linux__
 #include "config.h"
 
 #include <assert.h>
@@ -62,7 +61,9 @@ WINE_DEFAULT_DEBUG_CHANNEL(fsync);
 
 int fsync_enabled;
 static int current_pid;
-
+#ifndef __NR_futex
+#define __NR_futex 240
+#endif
 /* futex_waitv interface */
 
 #ifndef __NR_futex_waitv
@@ -1236,4 +1237,3 @@ NTSTATUS fsync_signal_and_wait( HANDLE signal, HANDLE wait, BOOLEAN alertable,
 
     return fsync_wait_objects( 1, &wait, TRUE, alertable, timeout );
 }
-#endif // __linux
-- 
2.39.5 (Apple Git-154)

