From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: mrvictory1 <108172910+mrvictory1@users.noreply.github.com>
Date: Sat, 4 Jul 2026 08:02:56 +0300
Subject: [PATCH 11/13] Define __NR_futex on server/fsync.c if missing

---
 server/fsync.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/server/fsync.c b/server/fsync.c
index 7d3ec646f9e..f7847bb49a3 100644
--- a/server/fsync.c
+++ b/server/fsync.c
@@ -51,7 +51,9 @@
 #ifndef __NR_futex_waitv
 #define __NR_futex_waitv 449
 #endif
-
+#ifndef __NR_futex
+#define __NR_futex 240
+#endif
 int do_fsync_cached = -1;
 
 int fsync_check_support(void)
-- 
2.39.5 (Apple Git-154)

