

Date: Wed, 22 Jul 2026 14:38:22 +0300
Subject: [PATCH 3/4] loader64/Makefile.in: don't depend on wine_info.plist

---
 loader64/Makefile.in | 2 --
 1 file changed, 2 deletions(-)

diff --git a/loader64/Makefile.in b/loader64/Makefile.in
index 1004f1967b5..62b0b5e4d2e 100644
--- a/loader64/Makefile.in
+++ b/loader64/Makefile.in
@@ -12,9 +12,7 @@ UNIX_CFLAGS = -fPIE
 preloader_EXTRADEFS = -fno-builtin
 
 wine64_OBJS = main.o
-wine64_DEPS = $(WINELOADER_DEPENDS)
 wine64_LDFLAGS = $(WINELOADER_LDFLAGS) $(LDEXECFLAGS) $(PTHREAD_LIBS)
 
 wine64_preloader_OBJS = preloader.o $(wine64_OBJS)
-wine64_preloader_DEPS = $(WINELOADER_DEPENDS)
 wine64_preloader_LDFLAGS = $(WINEPRELOADER_LDFLAGS)
-- 
2.50.1 (Apple Git-155)

