

Date: Thu, 30 Jul 2026 13:33:29 +0300
Subject: [PATCH 4/4] winnt.h: change types to make clang happy

---
 include/winnt.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/winnt.h b/include/winnt.h
index bdc86af59f2..24845957872 100644
--- a/include/winnt.h
+++ b/include/winnt.h
@@ -2513,7 +2513,7 @@ static FORCEINLINE struct _TEB * WINAPI NtCurrentTeb(void)
     return teb;
 }
 #elif defined(__i386__) && defined(_MSC_VER)
-DWORD __readfsdword(DWORD);
+unsigned long __readfsdword(unsigned long);
 #pragma intrinsic(__readfsdword)
 static FORCEINLINE struct _TEB * WINAPI NtCurrentTeb(void)
 {
@@ -7190,7 +7190,7 @@ static FORCEINLINE __int64 InterlockedAdd64( __int64 volatile *dest, __int64 val
 
 static FORCEINLINE void MemoryBarrier(void)
 {
-    LONG dummy;
+    volatile long dummy;
     InterlockedOr(&dummy, 0);
 }
 
-- 
2.50.1 (Apple Git-155)

