From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: mrvictory1 <108172910+mrvictory1@users.noreply.github.com>
Date: Tue, 14 Jul 2026 12:25:57 +0300
Subject: [PATCH 18/42] Makefile.in: disable 32-bit build of gst_libav

---
 Makefile.in | 1 -
 1 file changed, 1 deletion(-)

diff --git a/Makefile.in b/Makefile.in
index 38e32c59..72240d39 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -340,7 +340,6 @@ $(OBJ)/.ffmpeg-aarch64-dist:
 GST_LIBAV_DEPENDS = gst_orc gstreamer gst_base ffmpeg
 
 $(eval $(call rules-source,gst_libav,$(SRCDIR)/gstreamer/subprojects/gst-libav))
-$(eval $(call rules-meson,gst_libav,i386,unix))
 $(eval $(call rules-meson,gst_libav,x86_64,unix))
 $(eval $(call rules-meson,gst_libav,aarch64,unix))
 
-- 
2.50.1 (Apple Git-155)

