From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: mrvictory1 <108172910+mrvictory1@users.noreply.github.com>
Date: Tue, 14 Jul 2026 12:32:15 +0300
Subject: [PATCH 19/42] rules-cargo.mk: hardcode linker as gcc

---
 make/rules-cargo.mk | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/make/rules-cargo.mk b/make/rules-cargo.mk
index 776fb915..c70d92df 100644
--- a/make/rules-cargo.mk
+++ b/make/rules-cargo.mk
@@ -38,4 +38,5 @@ aarch64-unix_CARGO_ARGS := --target $(aarch64-unix_CARGO_TARGET)
 # Used in rules-common.mk
 # We need to specify the linker explicitly for at least --target architecture as
 # well as host's architecture - some crates build things on host via build.rs
-CARGO_LINKERS :=  $(foreach a,$(unix_ARCHS),CARGO_TARGET_$(call toupper,$($(a)-unix_CARGO_TARGET))_LINKER="$($(a)-unix_TARGET)-gcc")
+# macOS: ignore the above and hardcode as gcc
+CARGO_LINKERS :=  $(foreach a,$(unix_ARCHS),CARGO_TARGET_$(call toupper,$($(a)-unix_CARGO_TARGET))_LINKER="gcc")
-- 
2.50.1 (Apple Git-155)

