From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: mrvictory1 <108172910+mrvictory1@users.noreply.github.com>
Date: Thu, 30 Jul 2026 13:01:13 +0300
Subject: [PATCH 26/42] Makefile.in: disable some 32-bit dependencies &
 components

---
 Makefile.in | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/Makefile.in b/Makefile.in
index b3a1b1ab..3fc6bc23 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -418,7 +418,7 @@ $(eval $(call rules-cmake,glslang,aarch64,unix))
 LSTEAMCLIENT_DEPENDS = wine
 
 $(eval $(call rules-source,lsteamclient,$(SRCDIR)/lsteamclient))
-$(eval $(call rules-makedep,lsteamclient,i386))
+# $(eval $(call rules-makedep,lsteamclient,i386))
 $(eval $(call rules-makedep,lsteamclient,x86_64))
 $(eval $(call rules-makedep,lsteamclient,aarch64))
 
@@ -464,7 +464,7 @@ $(WINEOPENXR64_JSON): $(WINEOPENXR_SRC)/wineopenxr64.json
 STEAMEXE_DEPENDS = wine
 
 $(eval $(call rules-source,steamexe,$(SRCDIR)/steam_helper))
-$(eval $(call rules-makedep,steamexe,i386))
+# $(eval $(call rules-makedep,steamexe,i386))
 $(eval $(call rules-makedep,steamexe,x86_64))
 $(eval $(call rules-makedep,steamexe,aarch64))
 
@@ -685,7 +685,7 @@ endif
 VRCLIENT_DEPENDS = vulkan-headers wine
 
 $(eval $(call rules-source,vrclient,$(SRCDIR)/vrclient_x64))
-$(eval $(call rules-makedep,vrclient,i386))
+# $(eval $(call rules-makedep,vrclient,i386))
 $(eval $(call rules-makedep,vrclient,x86_64))
 $(eval $(call rules-makedep,vrclient,aarch64))
 
@@ -1446,14 +1446,14 @@ redist: all
 
 .PHONY: module32 module64 module
 
-module32: | all-source wine-i386-configure
+module32: | all-source wine-x86_64-configure
 	+$(MAKE) -j$(J) $(filter -j%,$(MAKEFLAGS)) $(MFLAGS) $(MAKEOVERRIDES) -C $(WINE_i386_OBJ)/dlls/$(module)
 
 module64: | all-source wine-x86_64-configure
 	+$(MAKE) -j$(J) $(filter -j%,$(MAKEFLAGS)) $(MFLAGS) $(MAKEOVERRIDES) -C $(WINE_x86_64_OBJ)/dlls/$(module)
 
 module: | all-source wine-configure
-module: module32 module64
+module: module64
 
 ###############################
 else # outside of the container
-- 
2.50.1 (Apple Git-155)

