From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: mrvictory1 <108172910+mrvictory1@users.noreply.github.com>
Date: Thu, 30 Jul 2026 13:16:30 +0300
Subject: [PATCH] Makefile.in: copy dependencies for macOS.

---
 Makefile.in | 15 +++++++++++++++
 1 file changed, 15 insertions(+)

diff --git a/Makefile.in b/Makefile.in
index 12c87cb5..75282939 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -1321,6 +1321,21 @@ $(OVR_aarch64_DIST): $(SRCDIR)/openvr/bin/win64/openvr_api.dll | $(DST_DIR)
 all-dist: $(OVR_aarch64_DIST)
 endif
 
+##
+## macos_deps
+##
+
+MACOS_DEPS_DIST := $(DST_DIR)/lib/wine/x86_64-unix/
+BREW_LIB := /usr/local/lib
+
+$(OBJ)/.macos_deps-x86_64-dist: wine-x86_64-build
+	cp -f $(BREW_LIB)/libfreetype.6.dylib $(MACOS_DEPS_DIST)
+	cp -f $(BREW_LIB)/libMoltenVK.dylib $(MACOS_DEPS_DIST)
+	cp -f $(BREW_LIB)/libgnutls.30.dylib $(MACOS_DEPS_DIST)
+	cp -f $(BREW_LIB)/libgmp.10.dylib $(MACOS_DEPS_DIST)
+	touch $@
+
+all-dist: $(OBJ)/.macos_deps-x86_64-dist
 
 ##
 ## default_pfx
-- 
2.50.1 (Apple Git-155)

